### ISSUE TITLE
能否将上报数据存在数据库或oss？

### ISSUE THREAD

**Issue by Drswith** [xy-sea/web-see#11]
跑web-see-demo的时候发现serveri.js里写的上报数据是放内存里的
如题，能否实现存储在数据库或oss？

其他数据还好，录屏的话数据类型是什么？只能看到event字段是很长的字符串类型

**Comment by codedogQBY** [comment]
录屏是压缩后的base64字符串吧

**Comment by xy-sea** [comment]
是的 上报的数据需要存到数据库中 我给出的serveri.js只是一个示例 用于演示录屏的播放 录屏的数据是一个很长的base64字符串

**Comment by GuangMingZ** [comment]
录屏能力可以自己集成rrweb，实现上报，web-see的代码逻辑很清晰，自己定制不会很难
